Signal/src/util/OWSBackupExportJob.m

```diff
@@ -294,6 +294,10 @@ @interface OWSBackupExportJob ()
 
 @property (nonatomic, nullable) OWSBackupExportItem *manifestItem;
 
+// If we are replacing an existing backup, we use some of its contents for continuity.
+@property (nonatomic, nullable) NSDictionary<NSString *, OWSBackupManifestItem *> *lastManifestItemMap;
+@property (nonatomic, nullable) NSSet<NSString *> *lastRecordNames;
```

Comment:
In order to do incremental backup, we need to try to "re-use/recycle" backup fragments from previous backups.  To do that, we need to:

* Defensively verify that recycled fragments still exist.
* Extract and reuse the fragment's metadata from the last backup.